tools/cpp/windows_cc_configure.bzl

```diff
@@ -229,32 +229,38 @@ def _is_vs_2017_or_2019(vc_path):
     # C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\
     return vc_path.find("2017") != -1 or vc_path.find("2019") != -1
 
-def _find_vcvarsall_bat_script(repository_ctx, vc_path):
-    """Find vcvarsall.bat script. Doesn't %-escape the result."""
+def _find_vcvars_bat_script(repository_ctx, vc_path):
+    """Find batch script to set up environment variables for VC. Doesn't %-escape the result."""
     if _is_vs_2017_or_2019(vc_path):
-        vcvarsall = vc_path + "\\Auxiliary\\Build\\VCVARSALL.BAT"
+        vcvars_script = vc_path + "\\Auxiliary\\Build\\VCVARSALL.BAT"
     else:
-        vcvarsall = vc_path + "\\VCVARSALL.BAT"
+        vcvars_script = vc_path + "\\bin\\amd64\\VCVARS64.BAT"
```

Comment:
We have to explicitly use `bin\amd64\VCVARS64.BAT` because the `VCVARSALL.BAT` in VC 2015 fall back to `vcbuildtools.bat` when VC 2017 or VC 2019 is installed. But `vcbuildtools.bat` doesn't support specifying Windows SDK. I also double checked, if no Windows SDK version is passed, `vcbuildtools.bat` sets the exactly same environment variable as `VCVARSALL.BAT`.

Reply:
Thanks for checking and for the comment!